sass/pages/_home.scss

```diff
@@ -1 +1,111 @@
-//This styles are specific styles for home.php
\ No newline at end of file
+//This styles are specific styles for home.php
+.latest_post{
+
+  @include pink_background_gradient;
+
+    .latest_post_wrapper{
+      padding: 3rem 1rem 2rem 1rem;
+
+      .latest_post_image{
+        @include image_box_shadow;
+      }
+    }
+  }
+
+  .post_content{
+    padding-top: 3rem;
+    padding-bottom: 1rem;
+  }
+
+  .all_posts{
+    background: linear-gradient(to right, $palepink_color, 5%, white);
+    padding-top: 2rem; 
+    padding-bottom: 2rem;
+  }
+
+  .blog_post{
+    margin: 1rem;
+  }
+  //Overlay effect on post images
+
+  .image {
+    display: block;
+    width: 100%;
+    height: auto;
+  }
+  
+  .overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: auto;
+    width: 100%;
+    opacity: 0;
+    transition: .5s ease;
+    background-color: $darkblue_color;
+  }
+  
+  .container:hover .overlay {
+    opacity: 0.9;
+  }
+  
+  .text {
```

Comment:
Text should be specified in typography-file, I think?

Reply:
This seems t obe a word like container in bootstrap for functionality, i could not change it with anything else, so I mada a comment. 
Whould you prefer I moved til to typography?

.text {
    font-size: 1.5rem;
    font-family: $serif;
    color: white;
    position: absolute;
    top: 50%;
    left: 50%;
    -webkit-transform: translate(-50%, -50%);
    -ms-transform: translate(-50%, -50%);
    transform: translate(-50%, -50%);
    text-align: center;
  }